applications/tasks/src/modules/kt_rates.erl

```diff
@@ -0,0 +1,333 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2013-2016, 2600Hz
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Sergey Korobkov
+%%%-------------------------------------------------------------------
+-module(kt_rates).
+%% behaviour: tasks_provider
+
+-export([init/0
+        ,help/1, help/2, help/3
+        ,output_header/1
+        ,cleanup/2
+        ]).
+
+%% Verifiers
+-export([direction/1
+        ]).
+
+%% Appliers
+-export([export/2
+        ,import/17
+        ,delete/17
+        ]).
+
+-include("tasks.hrl").
+
+-define(CATEGORY, "rates").
+-define(ACTIONS, [<<"export">>
+                 ,<<"import">>
+                 ,<<"delete">>
+                 ]).
+-define(RATES_VIEW, <<"rates/lookup">>).
+-define(BULK_LIMIT, 500).
+
+%% All this macros MUST be synchronized to each other
+-define(VARS, Prefix, Cost, RatedeckId, AccountId, ISO,
+        Desc, Name, Surcharge, Increment, Minimum,
+        Direction, IntCost, IntSurcharge, NoCharge, Weight).
+-define(SPEC_FIELDS, api_binary(), api_binary(), api_binary(), api_binary(), api_binary(),
+        api_binary(), api_binary(), api_binary(), api_binary(), api_binary(),
+        api_binary(), api_binary(), api_binary(), api_binary(), api_binary()).
+-define(DOC_FIELDS, <<"prefix">>, <<"rate_cost">>, <<"ratedeck">>, <<"account_id">>, <<"iso_country_code">>,
```

Comment:
I wonder if there's a way to generate this from the rates schema directly? I don't know that its a blocker for this PR but was something I was thinking about...